test/Microsoft.Azure.Devices.Edge.Test.Common/config/EdgeConfigBuilder.cs

```diff
@@ -0,0 +1,160 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Test.Common.Config
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using Microsoft.Azure.Devices.Edge.Util;
+
+    public class EdgeConfigBuilder
+    {
+        readonly string deviceId;
+        readonly Dictionary<string, IModuleConfigBuilder> moduleBuilders;
+        Option<(string address, string username, string password)> registry;
+
+        public EdgeConfigBuilder(string deviceId)
+        {
+            this.deviceId = deviceId;
```

Comment:
check empty for device id

Reply:
If callers shouldn't pass a meaningless argument to this constructor, then the call to `RegistryManager.ApplyConfigurationContentOnDeviceAsync` will fail later on. In this internal test code, that failure is good enough.